<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWidget</class>
 <widget class="QWidget" name="AddStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Name">
       <property name="text">
        <string>学生姓名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ID">
       <property name="text">
        <string>学生学号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Programming">
       <property name="text">
        <string>程序设计成绩</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="PE">
       <property name="text">
        <string>体育成绩</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name_Text"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ID_Text"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Programming_Text"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="PE_Text"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Math_Text"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="English_Text"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Math">
       <property name="text">
        <string>数学成绩</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="English">
       <property name="text">
        <string>英语成绩</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="Province"/>
     </item>
     <item>
      <widget class="QComboBox" name="City"/>
     </item>
     <item>
      <widget class="QComboBox" name="District"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>录入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
